lake-manifest.json

```diff
@@ -15,17 +15,17 @@
    "type": "git",
    "subDir": null,
    "scope": "",
-   "rev": "c44e0c8ee63ca166450922a373c7409c5d26b00b",
+   "rev": "c3b95a796220351ed0edb71bedd08a29f4a6de43",
    "name": "mathlib",
    "manifestFile": "lake-manifest.json",
-   "inputRev": "v4.19.0",
+   "inputRev": null,
```

Comment:
We should pin the new toolchain version.

Reply:
Should be always be pinning even if the version of mathlib which I bump to might not be the one which mathlib has tagged as `v4.20.0-rc5` and thus the repo might not compile with v4.20.0-rc5 of mathlib? I don't understand the point of pinning.

Reply:
Since automatic release tagging is enabled in FLT, this pull request will be tagged upon merging. The purpose of tagging commits is to ensure that the toolchain version of FLT remains compatible with the corresponding toolchain version of Mathlib, as well as with any other downstream projects that depend on Mathlib and might, at some point, also import FLT.

Reply:
But what I'm saying is that I am definitely *not* claiming that this PR is compatible with mathlib `commit 4e685d14db58b1ab29024bb0761902a3615f3f14 (tag: v4.20.0-rc5)` from two days ago, I am claiming that this PR is compatible with a later mathlib commit from today.

Reply:
Perhaps I am misunderstanding what this `inputRev` field is saying. Is it simply saying "I work with some unspecified commit of mathlib which works under Lean `v4.20.0-rc5`" (true) or is it saying "I work with the version of Lean tagged `v4.20.0-rc5` and the version of mathlib tagged `v4.20.0-rc5`" (probably false)?

Reply:
It says 

> I work with the version of Lean tagged `v4.20.0-rc5` and the version of mathlib tagged `v4.20.0-rc5`. 

So if we use automatic  release tagging we are forced to pin it with `rev = "v4.20.0-rc5"` before fixing all the bump-related issues.

Reply:
Right. On the other hand I am reluctant to go backwards now. So it looks like my response to #471 was incorrect: I should have first bumped to the mathlib tag (and then we would have been able to tag) and then to current master.

Reply:
I have tested it locally. It compiles without any further changes.

Reply:
Sure! Thanks for checking! I just couldn't face undoing changes which I'd already made but it sounds like there aren't any!

Reply:
Done.